survey_app/tests/test_views.py

```diff
@@ -46,7 +47,7 @@ def test_list_survey_questions(self):
 class QuestionOptionsViewTestCase(TestCase):
```

Comment:
Please research about APITestCase and update in your test cases @newDaria

Reply:
> Please research about APITestCase and update in your test cases @newDaria

ok